src/structure/pure/page.js

```diff
@@ -1,20 +1,30 @@
-/* page.js (c) 2012 by Christian Mayer [CometVisu at ChristianMayer dot de]
- *
+/* page.js 
+ * 
+ * copyright (c) 2010-2016 by Christian Mayer (ChristianMayer) [CometVisu at ChristianMayer dot de]
+ * 
  * This program is free software; you can redistribute it and/or modify it
  * under the terms of the GNU General Public License as published by the Free
  * Software Foundation; either version 3 of the License, or (at your option)
  * any later version.
  *
  * This program is distributed in the hope that it will be useful, but WITHOUT
  * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
- * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
  * more details.
  *
  * You should have received a copy of the GNU General Public License along
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA
+ *
+ * @module Page 
+ * @title  CometVisu Page 
  */
 
+
+/**
+ * @author Christian Mayer
+ * @since 2012
```

Comment:
Quite often 2012 is used in the files, but the content is older, most likely from 2010. If that's relevant I guess we need to dig a bit in the older SVN code...